gcloud/bigtable/happybase/connection.py

```diff
@@ -265,6 +269,77 @@ def tables(self):
 
         return table_names
 
+    def create_table(self, name, families):
+        """Create a table.
+
+        .. warning::
+
+            The only column family options from HappyBase that are able to be
+            used with Cloud Bigtable are ``max_versions`` and ``time_to_live``.
+
+        .. note::
+
+            This method is **not** atomic. The Cloud Bigtable API separates
```

Comment:
This is concerning. Can we provide any recommendations for recourse for the user? Or is it expected that they'll know how to deal with the broken state?